ibis/tests/all/test_string.py

```diff
@@ -112,7 +112,8 @@ def test_string_col_is_unicode(backend, alltypes, df):
         param(
             lambda t: t.string_col.length(),
             lambda t: t.string_col.str.len().astype('int32'),
-            id='length'
+            id='length',
+            marks=pytest.mark.xfail,
```

Comment:
This cannot be marked `xfail`. Does mapd not implement this operation?

Reply:
At this moment mapd doesn't support thinks like "SELECT 1;" or "SELECT COS(1);" in other word without the table identification.